src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -96,24 +34,94 @@ const defaultProps = {
     isVisible: false,
 };
 
-const ReportActionContextMenu = (props) => {
-    const wrapperStyle = getReportActionContextMenuStyles(props.isMini);
-    return props.isVisible && (
-        <View style={wrapperStyle}>
-            {CONTEXT_ACTIONS.map(contextAction => contextAction.shouldShow && (
-                <ReportActionContextMenuItem
-                    icon={contextAction.icon}
-                    text={contextAction.text}
-                    successIcon={contextAction.successIcon}
-                    successText={contextAction.successText}
-                    isMini={props.isMini}
-                    onPress={() => contextAction.onPress(props.reportAction)}
-                    key={contextAction.text}
-                />
-            ))}
-        </View>
-    );
-};
+class ReportActionContextMenu extends React.Component {
+    constructor(props) {
+        super(props);
+
+        /**
+         * A list of all the context actions in this menu.
+         */
+        this.CONTEXT_ACTIONS = [
```

Comment:
Since this isn't just a regular constant anymore and is now a class field, I think we should rename this in `camelCase`:

```suggestion
        this.contextActions = [
```

Reply:
bump